### PR TITLE
[Update] Add Swift package instructions

### PR DESCRIPTION
**Pull request by yo1995**
## Description

This PR adds Swift package instructions and changes the project to use the GitHub repo SP instead of local package.

## How To Test

As the https://github.com/Esri/arcgis-maps-sdk-swift package isn't public yet, you cannot build the project.

- Either test with the local packages by dragging in the toolkit and ArcGIS folder
- Or quickly test after the new SDK is released and report any issue it might cause
- Or use commit [f926975](https://github.com/Esri/arcgis-maps-sdk-swift/pull/2/commits/f92697566858be47b869d69e1be7fb8ff16480b9) if you know what you are doing, with https://github.com/Esri/arcgis-maps-sdk-swift/pull/2 😉

### CODE REVIEW

File: Shared/Supporting Files/Views/AboutView.swift [hunk 1]
```diff
@@ -58,7 +58,7 @@ private struct VersionRow: View {
 }
 
 private extension Bundle {
-    static let arcGIS = Bundle(identifier: "ArcGIS")!
+    static let arcGIS = Bundle(identifier: "com.esri.ArcGIS")!
```
Comment (yo1995) [inline Shared/Supporting Files/Views/AboutView.swift:61]: This line will crash if you use the local package, it is expected to work with the GitHub repo.

**Comment by yo1995** [comment]
Changing the toolkit dependency from local to remote will result in this warning.

image

**Comment by dfeinzimer** [comment]
> Changing the toolkit dependency from local to remote will result in this warning.

I believe that should drop off after release time on Wednesday

**Comment by zkline101** [comment]
> I believe that should drop off after release time on Wednesday

Yeah, that is right. Once this [PR](https://github.com/Esri/arcgis-maps-sdk-swift/pull/2) is merged and then we add the tag and release.

File: README.md [hunk 2]
```diff
@@ -10,13 +10,15 @@ This repository contains Swift sample code demonstrating the capabilities of [Ar
...
-## Building Samples Using Swift API Manually 
+## Building Samples Using Swift Package Manager
 
 1. **Fork** and then **clone** the repository
-1. **Configure** the Swift API locally following the instructions in internal Swift API repo, section `Adding the ArcGIS library to your App`
 1. **Open** the `Samples.xcodeproj` **project** file
+    > The project has been configured to use the arcgis-maps-sdk-swift-toolkit package, which provides the ArcGISToolkit framework as well as the ArcGIS framework.
 1. **Run** the `Samples` app target
 
+> To add the Swift packages to your own projects, consult the documentation for the [ArcGIS Runtime Toolkit for Swift](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit#swift-package-manager) and [ArcGIS Maps SDK for Swift](https://github.com/Esri/arcgis-maps-sdk-swift#instructions).
```
Comment (yo1995) [inline README.md:20]:
```suggestion
> To add the Swift packages to your own projects, consult the documentation for the [ArcGIS Maps SDK for Swift Toolkit](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit#swift-package-manager) and [ArcGIS Maps SDK for Swift](https://github.com/Esri/arcgis-maps-sdk-swift#instructions).
```

**Review by zkline101 — APPROVED** [review]
Looks good. I tested with the final version of the SDK and I do see a few warnings:

Screenshot 2022-12-12 at 2 40 08 PM

**Comment by yo1995** [comment]
> Looks good. I tested with the final version of the SDK and I do see a few warnings:

Will fix in a separate PR, #102

**Review by sbaskaran — APPROVED** [review]

**Review by dfeinzimer — APPROVED** [review]

**Pull request merged**